### ISSUE TITLE
3.1.0: Null from typography on disable title

### ISSUE THREAD

**Issue by arberg** [henrytao-me/android-md-core#100]
supportActionBar.setDisplayShowTitleEnabled(false)

causes exception below. Reverting from 3.1.0 to 3.0.0 removes the issue.

I'm guessing its a missing null-check in MdToolbar.setTitle.

PS: gradle thinks version 23 is newer than version 3.0.0, its quite confusing, but is suppose its might be problematic to remove the 23.x.x versions.

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{dk.bnr.bnr/dk.bnr.androidbooking.MenuSettingsActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'int java.lang.CharSequence.length()' on a null object reference
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2509)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2569)
    at android.app.ActivityThread.access$900(ActivityThread.java:150)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1399)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:168)
    at android.app.ActivityThread.main(ActivityThread.java:5885)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:819)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:709)
 Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'int java.lang.CharSequence.length()' on a null object reference
    at android.text.SpannableString.<init>(SpannableString.java:30)
    at me.henrytao.mdcore.utils.Typography.applyTypeface(Typography.java:58)
    at me.henrytao.mdcore.widgets.internal.MdToolbar.setTitle(MdToolbar.java:60)
    at android.support.v7.widget.ToolbarWidgetWrapper.setDisplayOptions(ToolbarWidgetWrapper.java:402)
    at android.support.v7.app.ToolbarActionBar.setDisplayOptions(ToolbarActionBar.java:264)
    at android.support.v7.app.ToolbarActionBar.setDisplayShowTitleEnabled(ToolbarActionBar.java:284)
    at dk.bnr.androidbooking.MenuSettingsActivity.onCreateSubActivity(MenuSettingsActivity.java:199)
    at dk.bnr.androidbooking.LoggedSubActivity.onCreate(LoggedSubActivity.java:22)
    at android.app.Activity.performCreate(Activity.java:6262)
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1125)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2462)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2569) 
    at android.app.ActivityThread.access$900(ActivityThread.java:150) 
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1399) 
    at android.os.Handler.dispatchMessage(Handler.java:102) 
    at android.os.Looper.loop(Looper.java:168) 
    at android.app.ActivityThread.main(ActivityThread.java:5885) 
    at java.lang.reflect.Method.invoke(Native Method) 
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:819) 
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:709) 
```

**Comment by henrytao-me** [comment]
Hi @arberg 

It's fixed in `3.1.1`. Thanks for your report. If you need further assist, feel free to ask me. If you think this library is useful, please spread it out. Thanks and happy coding. 

Cheers, 
Henry

**Comment by arberg** [comment]
Thank you. best Alex
